readme_generator/nginx.conf

```diff
@@ -1,17 +1,17 @@
 location /github {
 
-  # Force usage of https
-  if ($scheme = http) {
-    rewrite ^ https://$server_name$request_uri? permanent;
-  }
+    # Force usage of https
+    if ($scheme = http) {
+        rewrite ^ https://$server_name$request_uri? permanent;
+    }
 
-  client_max_body_size 100M;
+    client_max_body_size 100M;
 
-  proxy_pass http://127.0.0.1:8123;
-  proxy_http_version 1.1;
-  proxy_set_header Upgrade $http_upgrade;
-  proxy_set_header Connection "Upgrade";
+    proxy_pass http://127.0.0.1:8123;
+    proxy_http_version 1.1;
+    proxy_set_header Upgrade $http_upgrade;
+    proxy_set_header Connection "Upgrade";
 
-  # preserve client IP
-  proxy_set_header X-Forwarded-For $remote_addr;
+    # preserve client IP
+    proxy_set_header X-Forwarded-For $remote_addr;
```

Comment:
Here is the tricky part.

nginx conf files seem to have 2-space indentation code style.

I don't know what to do with that: should it remain consistent with the rest of the files (4-spaces indentation) or should it be an exception (2-spaces)?

Reply:
In most of my apps I have 4 space indentation in nginx config so I suppose the nginx config indentation seem not really consistant. From my point of view we can make this consistent with the rest with 4 space and we can fix the place where it's not the case. But we might need the point of view from other packager.